tslint.json

```diff
@@ -102,6 +102,7 @@
       "check-separator",
       "check-type"
     ],
-    "jsx-no-multiline-js": false
+    "jsx-no-multiline-js": false,
+    "jsx-no-lambda": false
```

Comment:
I vote we stop enforcing `jsx-no-lambda`.  come fight me.

Reply:
:roll_over_immediately:

Reply:
This seems okay if we agree to only use lambdas for `onChange`/`onClick` handlers, maybe?

If we are passing them around as props, that's not great (from what I've read), as they'll always be different and potentially trigger unnecessary `render` calls.

Reply:
> as they'll always be different and potentially trigger unnecessary render calls.

This is only true for `PureComponent` (which checks props for equality, and avoids calling render when nothing has changed).  Normal `Component`s on the other hand will still re-`render` even if all the props are equal (and we don't use `PureComponent`)

https://cdb.reacttraining.com/react-inline-functions-and-performance-bdff784f5578#2fc6

So I'm convinced the "no lambda" rule is mostly cargo cult, and definitely a premature optimization.  

That said, I generally want to use them when a callback requires parameters that come from a looped element such as 
```jsx
{ items.map(item => (
  <div onClick={() => this.handleClick(item)} />
))}
```